css/style.css

```diff
@@ -410,14 +417,15 @@ body {
     -webkit-box-pack: center;
     -ms-flex-pack: center;
     justify-content: center;
-    width: 488px;
+    max-width: 488px;
     padding: 0;
+     margin: 0 auto;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/chendler_tesla/pullRequest?prid=5708338)